scripts/centos/cleanup.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh -eux
 
 if [ -s /etc/oracle-release ]; then
-  distro = 'oracle'
+  distro='oracle'
```

Comment:
It is bash :wink: And also: I did not write this – this is synced from the Bento repository :wink: